OLD/ocrolib/ngraphs.py

```diff
@@ -106,11 +108,11 @@ def __init__(self,*args,**kw):
         NGraphsCounts.__init__(self,*args,**kw)
     def buildFromFiles(self,fnames,n):
         """Given a set of files, build the log posteriors."""
-        print "reading",len(fnames),"files"
+        print("reading", len(fnames), "files")
         counter = self.computeNGraphs(fnames,n)
-        print "got",sum(counter.values()),"%d-graphs"%(n,)
+        print("got", sum(counter.values()), "%d-graphs" % (n, ))
```

Comment:
This looks strange: After the last comma there is nothing but the closing parenthesis.